js/windowMgr.js

```diff
@@ -73,6 +73,7 @@ function doCreateMainWindow(initialUrl, initialBounds) {
         show: true,
         minWidth: MIN_WIDTH,
         minHeight: MIN_HEIGHT,
+        titleBarStyle: 'hidden',
```

Comment:
I have added this ```-webkit-app-region: drag``` class to the header section in client-app which takes care of the draggable window.